Sources/MapboxNavigation/NavigationOptions.swift

```diff
@@ -47,6 +47,12 @@ open class NavigationOptions: NavigationCustomizable {
      */
     open var predictiveCacheOptions: PredictiveCacheOptions?
```

Comment:
And, by the way, we have `...Options` semantic here. Maybe it's better to keep this in this PR and rename `TileStoreConfiguration` into `TileStoreOptions`?

Reply:
Unfortunately it will cause naming conflict with Common's `MBXTileStoreOptions`. I would like to avoid namespace usage.

Reply:
I agree that “options” would be a more consistent naming choice. Internal type conflicts ideally shouldn’t influence what we expose publicly to developers, but this would be an opportunity to prefer the more grammatical “TileStorage” terminology over “TileStore” (which incidentally sounds like [a kind of shop](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dtiles)).